site/src/util/workspace.test.ts

```diff
@@ -40,4 +40,16 @@ describe("util > workspace", () => {
       expect(isWorkspaceOn(workspace)).toBe(isOn)
     })
   })
+  describe("workspaceQueryToFilter", () => {
+    it.each<[string | undefined, TypesGen.WorkspaceFilter]>([
+      [undefined, {}],
+      ["", {}],
+      ["asdkfvjn", { name: "asdkfvjn" }],
+      ["owner:me", { owner: "me" }],
+      ["owner:me owner:me2", { owner: "me" }],
+      ["me/dev", { owner: "me", name: "dev" }],
```

Comment:
```suggestion
      ["me/dev", { owner: "me", name: "dev" }],
      ["  owner:me       key:val    ", { owner: "me" }],
```

To test the whitespace preparing/replacement logic I added